waltid-services/waltid-wallet-api/src/test/kotlin/E2EWalletTestLocal.kt

```diff
@@ -17,19 +14,9 @@ import io.ktor.client.request.*
 import io.ktor.http.*
 import io.ktor.serialization.kotlinx.json.*
 import io.ktor.server.testing.*
-import kotlinx.coroutines.test.runTest
-import kotlinx.serialization.json.JsonArray
-import kotlinx.serialization.json.jsonObject
-import kotlinx.serialization.json.jsonPrimitive
-import kotlinx.uuid.UUID
 import kotlin.io.path.Path
 import kotlin.io.path.absolutePathString
 import kotlin.test.Test
-import kotlin.test.assertNotEquals
-import kotlin.time.Duration.Companion.seconds
-import id.walt.issuer.base.config.ConfigManager as IssuerConfigManager
-import id.walt.webwallet.config.ConfigManager as WalletConfigManager
-import id.walt.webwallet.config.WebConfig as WalletWebConfig
 
 class E2EWalletTestLocal : E2EWalletTestBase() {
```

Comment:
since we're moving to a new e2e test structure, maybe this can be removed and have the e2e tests added with another pr?